internal/util/logger.go

```diff
@@ -0,0 +1,50 @@
+package util
+
+import (
+	"log/slog"
+	"net/http"
+	"os"
+)
+
+func NewLogger(serviceName string) *slog.Logger {
```

Comment:
Is there a reason why these have been copied from opg-go-common instead of using the telemetry package in the common module?

Reply:
The version of opg-go-common that I'd upgraded to doesn't have that package, but that sounds much better - I'll upgrade it and use that